src/platformdirs/macos.py

```diff
@@ -1,9 +1,20 @@
 import os
 
 from .api import PlatformDirsABC
+from .unix import Unix
 
+XDG_SUPPORT = {
```

Comment:
Why do we need this map, in the unix we achieve it without doing this :thinking:

Reply:
It's there for the purposes of supporting testing. It's used to determine which compat tests to skip for macOS because of the introduction of XDG.

Reply:
Let's remove it. Tests should not really on globals from modules, otherwise a typo here would never be caught potentially.